dom.bs

```diff
@@ -6697,23 +6715,19 @@ method, when invoked, must run these steps:
 
 <pre class=idl>
 [Exposed=Window]
-interface Attr {
+interface Attr : Node {
   readonly attribute DOMString? namespaceURI;
   readonly attribute DOMString? prefix;
   readonly attribute DOMString localName;
```

Comment:
The reason I am asking is that WebKit has these 3 on Node, instead of duplicating them between Attr and Element.